frontend/src/App.scss

```diff
@@ -29,3 +41,28 @@ body {
     border-color: #545b62;
   }
 }
+:focus {
+  outline: 0 !important;
```

Comment:
Usually not a good idea to use `!important` in CSS properties because it usually means you can edit the property in a different part of the UI. We can fix this in a future PR though.